xbmc/windowing/X11/WinSystemX11GLContext.cpp

```diff
@@ -285,8 +285,7 @@ bool CWinSystemX11GLContext::RefreshGLContext(bool force)
         X11::VaapiProxyConfig(m_vaapiProxy.get(), GetDisplay(),
                               static_cast<CGLContextEGL*>(m_pGLContext)->m_eglDisplay);
         bool general, deepColor;
```

Comment:
Shouldn't the variables set to false here?
Otherwise the `#if !defined(HAVE_LIBVA)` version of `X11::VAAPIRegisterRender` has to set them to false too

Reply:
One could do that, but only `general` needs to be initialized, right? If `!general`, then `deepColor` should not ever be evaluated.